main/common/settings.js

```diff
@@ -90,12 +92,16 @@ const audioInputDeviceId = store.get('audioInputDeviceId');
 (async () => {
   const devices = await audioDevices();
 
+  if (!Array.isArray(devices)) {
+    const Sentry = require('../utils/sentry');
+    Sentry.captureMessage(`devices is not a string: ${JSON.stringify(devices)}`);
```

Comment:
You mean not an array?

```suggestion
    Sentry.captureMessage(`devices is not an array: ${JSON.stringify(devices)}`);
```

Reply:
🤦‍♂ Yes